### DISCUSSION TITLE
Epub export - Linux system (ubuntu)

### DISCUSSION THREAD

**Discussion by RafaDarque** [vkbo/novelWriter#1115] [Help] [answered]
Hello. I installed novelWriter to start writting my book. I am really enjoying the interface, however i really wanted to know if there is a way to export directly to epub. Plus, is it possible to export as a5, instead of a4? Thank you!

**Comment by vkbo** [comment, marked as answer]
Hi!

No, there is no epub export. I've considered it, but there are several tools that can do this already, with a lot of options for customisation. For instance [Calibre](https://calibre-ebook.com/). There are also other tools. I may add support for epub later on, but novelWriter is intended to produce a manuscript, not a final product, so it hasn't been priority.

As for the paper format, novelWriter doesn't specify one at all. If you're referring to the Open Document export format, it is your word processor that picks A4, as it is likely the default paper format for your system. You should be able to change it in the word processor.

If you need to change format when making a PDF, use the "Print Preview" option instead. Then select the print icon, and select the PDF printer. There should be a properties button that lets you change format. The "Print to PDF" export option in the Build Tool is a shortcut for the same options, but it uses the defaults.

**Reply by RafaDarque** [reply]
Thank you so much for your answer. And by the way, congrats on the software. Finally found one that i really like! Keep with the good work!

**Reply by vkbo** [reply]
Thanks! I'm glad you find it useful :)
